### ISSUE TITLE
entering user name for comment in validation

### ISSUE THREAD

**Issue by johnwhelan** [hotosm/tasking-manager#894]
TM2 would suggest them or you could cut and paste TM3 you have to type them in.

**Comment by alekno** [comment]
Why cut & paste? What is the difference to mechanical spam, when doing mass c&p? I have seen some validators overdoing personal messaging. Quite annoying, if it is not a specific and unique message.

**Comment by johnwhelan** [comment]
When I validate a tile not all the mapping has been done by the person who marked the tile done.  At times I want to leave a message to someone else with a relevant pointer to the wiki and I've picked up their name with a <crtl>i or <crtl>h in JOSM.  <crtl>v works nicely in this case.  Again I may wish to leave a message to the mapper.  Other than one to Blake I have not left a "Hi Johnwhelan I just validated your mapping on Task 33. Awesome work! Keep mapping :): " message to any of the mappers I've validated.
